workers/generic-worker/chain_of_trust_posix.go

```diff
@@ -1,4 +1,4 @@
-// +build multiuser,linux multiuser,darwin
+// +build multiuser,linux multiuser,darwin,freebsd
```

Comment:
This change isn't needed - chain of trust isn't supported in simple engine.

Reply:
Okay thank you for the clear explanation.  I looked at my test script and wasn't building multiuser.  I've changed it now to just use `build.sh -a` now for testing before submission sorry for the trouble.